### PR TITLE
Add Gov-Client-Browser-JS-User-Agent header to HMRC

### PR DESCRIPTION
**Pull request by rachanamamillapalli**
## Description of what's changing
Add Gov-Client-Browser-JS-User-Agent header to HMRC

## Which issue does this PR relate to?
Solves #124 

## Checklist

[ ] Tests are written and maintain or improve code coverage
[ ] I've tested this in my application using `yarn link` (if applicable)
[ ] You consent to and are confident this change can be released with no regression

### CODE REVIEW

File: src/js/common/browserInfoHelper.js [hunk 1]
```diff
@@ -169,3 +169,11 @@ export const getBrowserDoNotTrackStatus = () => {
       windowVar.external.msTrackingProtectionEnabled());
   return isBrowserDoNotTrack ? "true" : "false";
 };
+
+/**
+ * This function returns user agent by checking the navigator
+ * @returns {String} user agent for the current browser
+ */
+export const getUserAgent = () => {
+  return navigator.userAgent
```
Comment (skodamarthi) [inline src/js/common/browserInfoHelper.js:178]: semicolon at the end to be consistent with the style we are using in this repo.
```suggestion
  return navigator.userAgent;
```

File: tests/unit/common/browserInfoHelper.test.js [hunk 2]
```diff
@@ -265,6 +265,14 @@ describe("BrowserInfoHelper", () => {
     expect(await getDeviceLocalIPAsString()).toEqual({"deviceIpString": "127.0.0.1", "deviceIpTimeStamp": mockTimeStamp});
   });
 
+  it("getUserAgent", async () => {
+    const mockedUserAgent = "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/94.0.4606.61 Safari/537.36"
+    navigatorSpy.mockImplementation(() => ({
+      userAgent: mockedUserAgent
+    }));
+    expect(getUserAgent()).toEqual(mockedUserAgent)
```
Comment (skodamarthi) [inline tests/unit/common/browserInfoHelper.test.js:273]:
```suggestion
    expect(getUserAgent()).toEqual(mockedUserAgent);
```

File: tests/unit/common/browserInfoHelper.test.js [hunk 3]
```diff
@@ -265,6 +265,14 @@ describe("BrowserInfoHelper", () => {
     expect(await getDeviceLocalIPAsString()).toEqual({"deviceIpString": "127.0.0.1", "deviceIpTimeStamp": mockTimeStamp});
   });
 
+  it("getUserAgent", async () => {
+    const mockedUserAgent = "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/94.0.4606.61 Safari/537.36"
```
Comment (skodamarthi) [inline tests/unit/common/browserInfoHelper.test.js:269]:
```suggestion
    const mockedUserAgent = "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/94.0.4606.61 Safari/537.36";
```

**Review by skodamarthi — APPROVED** [review]

**Comment by rachanamamillapalli** [comment]
@skodamarthi Mind adding `hacktoberfest-accepted` label on the merged PR?

**Pull request merged**